Dockerfile

```diff
@@ -18,10 +18,10 @@ RUN pip install git+https://github.com/data-8/gitautosync && \
     jupyter serverextension enable --py nbgitpuller && \
     conda install -y -q nbval
 
-# create a python2 environment (for OMERO-PY compatibility)
+# create a python3 environment (for OMERO-PY compatibility)
```

Comment:
Since we've switched to Python 3 it might be possible to use the default jupyter environment (should lead to a smaller image) instead of creating a separate one. Depends on how this is used later.

Reply:
That is a bit unknown so I kept a separate one

Reply:
It will also probably depend of the python 3 version we support and the one installed
so far I went for 3.6

Reply:
OK, can always investigate in a separate PR.